### DISCUSSION TITLE
Some questions or improvements regarding the setting options

### DISCUSSION THREAD

**Discussion by MisakaSakura** [LizardByte/community#478] [Sunshine]
I have 2 questions.
First, why do I update the latest pre-release version, and I understand that the installation requires me to uninstall the old version, but why is my old version of the data also deleted?
As a result, I had to spend a lot of time and effort reconfiguring Sunshine again, and I didn't find a similar option in Sunshine settings to back up my settings;
Second, I don't need to use sunshine services all the time, so I don't want sunshine to boot automatically, but there is no option to turn off sunshine auto-start in the settings, and I can't find sunshine's boot service in Windows startup management.

**Comment by ReenigneArcher** [comment]
Did you manually put your own files inside the installation directory?

**Reply by MisakaSakura** [reply]
No, I didn't do anything, I just upgraded the program like the first install, like I said, I didn't know that a direct upgrade would overwrite or lose the configuration (because the other software I used, whether it was installed or the green portable version, could be upgraded directly to the overwrite without losing data), and the other reason was that I didn't know where the configuration file of Sunshine was.

**Reply by ReenigneArcher** [reply]
If you use the installer it should keep your config in place for updates. I do this several times a week for testing new builds, and my config is always left in place... but that only holds true if you are using the installer consistently. If you're are switching between portable and installer, then you won't have a great experience.

**Reply by MisakaSakura** [reply]
No, I just mentioned the installation and use of both software when I explained it to you, I'm not going to suddenly switch from the portable version to the installer and vice versa; as for sunshine, I've been using the installer version only, and updating with the installer as well, which is why I'm questioning you about the fact that my data from the old version wasn't preserved!

**Reply by ReenigneArcher** [reply]
What specifically was overwritten? Your apps file config?

**Reply by MisakaSakura** [reply]
I'm guessing it's all of them, because after I update it's just like installing sunshine for the first time, I need to start by creating a local account of my own for sunshine, and all the options in the settings are initialized and configured as well.

**Comment by reloxx13** [comment]
It always kept all configurations with the installer and by uninstalling with the installer. Did you uninstall yourself and not with the installer?

When you use the pre-release version you should always keep a backup in case of bugs which delete configs or other stuff, and overall you should have a backup of your pc or at least important files.

**Reply by MisakaSakura** [reply]
When I installed the new pre-release version I was prompted to uninstall the old version I just followed the instructions and used the installation guide of the upgrade package to complete the uninstallation of the old version to install the new version, I didn't think much of it because you know, for example, qbittorrent enhanced also uninstalls the old version for the user and then installs the new version, but it doesn't delete the user's data of the old version, and I thought that sunshine would not delete the user's old version. I didn't think that sunshine would do the same, but in fact my old version was cleared; you mentioned that it's a good idea to take a backup when updating the preview, but first of all sunshine doesn't provide a backup option, and I'm not sure where sunshine's configuration file is located.

**Reply by reloxx13** [reply]
I also press uninstall, and it did not delete the user data. It's strange it did for you, you may try it out again now and do a backup before. 
on my machine, the configs are in `C:\Program Files\Sunshine\config`.

**Reply by MisakaSakura** [reply]
Now I know that the default configuration file is in the location you mentioned, but this is still not intuitive and convenient for the average user, so I would like sunshine itself to provide import/export functionality!

[a comment by an automated account is not shown]
